packages/scroller/src/vaadin-scroller.js

```diff
@@ -15,12 +16,18 @@ import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mix
  *   <div>Content</div>
  * </vaadin-scroller>
  * ```
+ * The following attributes are exposed for styling:
+ *
+ * Attribute    | Description
+ * -------------| -----------
+ * `focus-ring` | Set when the element is focused using the keyboard.
```

Comment:
Should we also document `focused` attribute?

Reply:
Done

Reply:
The `focused` attribute is missing in the `.js` file

Reply:
Indeed, added